Source/WebKit/Modules/Internal/WebKitInternalCxx.h

```diff
@@ -25,4 +25,5 @@
 
 // Add project-level C++ header files here to be able to access them from within Swift sources.
 
+#import <UIProcess/SwiftDemoLogoConfirmation.h>
```

Comment:
Interesting that this is a bracket-style import, since it's not a public/private API header. Does it not work as a quote import, and if so why?

Reply:
Fixed.